src/helpers.rs

```diff
@@ -75,12 +74,15 @@ const UNIX_IO_ERROR_TABLE: &[(&str, std::io::ErrorKind)] = {
         ("ETIMEDOUT", TimedOut),
         ("ETXTBSY", ExecutableFileBusy),
         ("EXDEV", CrossesDevices),
-        // The following have two valid options. We have both for the forwards mapping; only the
-        // first one will be used for the backwards mapping.
+        // The following have multiple valid options. We have all of them for the forwards mapping;
+        // only the first one will be used for the backwards mapping.
         ("EPERM", PermissionDenied),
         ("EACCES", PermissionDenied),
         ("EWOULDBLOCK", WouldBlock),
         ("EAGAIN", WouldBlock),
+        ("ENOTSUP", Unsupported),
+        ("EOPNOTSUPP", Unsupported),
+        ("ENOSYS", Unsupported),
```

Comment:
Previously we had `Unsupported` map back t `ENOSYS`. This changes that to `ENOTSUP`. Why is that the better choice?